update.php

```diff
@@ -430,3 +430,18 @@ function update_1330()
 
 	return Update::SUCCESS;
 }
+
+function update_1332()
+{
+	$condition = ["`is-default` IS NOT NULL"];
+	$profiles = DBA::select('profile', [], $condition);
+
+	while ($profile = DBA::fetch($profiles)) {
```

Comment:
How many profile fields do you expect to update at large nodes? Will this work fast enough? I had this discussion with @annando in my last PR, so I've to ask :D

Reply:
Biggest Friendica node is about 10,000 users, with little more profiles than that. Depending on whether the profiles are completely filled or not, as many custom profile fields are created. I expect an average of 2-3 profile fields to be created per user.

Reply:
Stall I wait die @annando or merge it? :)